### ISSUE TITLE
Smart playlist support

### ISSUE THREAD

**Issue by benjaminlewandowski** [sefrijn/traktor-library-manager#4]
Looks like currently there is no smart playlist support / editor

**Comment by sefrijn** [comment]
Yeah, this could be a major update/improvement, that would also require quite some work I think.

**Comment by benjaminlewandowski** [comment]
kinda critical: existing smart playlist get there queries removed

i am working around that bug atm with reimporting a empty collection with just the smart playlists which "fixes" them
